docs/control/administering/debugging.rst

```diff
@@ -20,7 +20,7 @@ Logs from the Docker containers can be viewed using `the Docker CLI <https://doc
 Ubuntu
 ^^^^^^
 
-XXX This should be documented, see :issue:`1877`.
+Logs from the Flocker processes running on the nodes can be found in the :file:`var/log/flocker` directory.
 
```

Comment:
So right now these docs are really CentOS based, when a lot of the actions are not specifically CentOS actions below. At the very least this block should have a comment like:

`ubuntu$ cat /var/log/flocker/flocker-dataset-agent.log` is the ubuntu equivalent of `centos$ journalctl --all --output cat -u flocker-dataset-agent`.

A more complete solution would be to put these in environment variables, set them up 1 way for ubuntu, a different way for CentOS, and then have a new section for processing.  Right now that's not perfect though, because we seem to sort of willy-nilly add `--priority=err` to some of the commands below but not all :/ I'll leave it up to you if you want to do a big refactor or just add the one line, but I think we should at least give our ubuntu users something more.